docs/tsconfig.json

```
@@ -22,6 +22,9 @@
             "~flux/statistics/*": [
                 "/Users/bas/Development/Projects/flux/docs/node_modules/@flux-ui/statistics/src/*"
             ],
+            "~flux/flow/*": [
```

Biome (2.5.3)

[error] The key ~flux/flow/* was already declared.

(lint/suspicious/noDuplicateObjectKeys)